showpass/Showpass-E2E-tests
# TA-6-'Password Reset' modal window overall appearance

**Description:**
The following elements on the ‘Password Reset’ modal window will be verified:
- ‘Showpass’ logo
- ‘Password Reset’ header
- ‘Forgot your password? No worries! We'll send you a link to reset it, just enter your email below.’ text
- ‘Email Address’ label and input field
- ‘Send Reset Link’ button
- ‘I remember my password!’ button

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-6

**How to test:**
Run the following spec and ensure it passes
cypress/e2e/home/home-page-appearance.cy.js

**Test execution results:**

<img width="1032" alt="TA-6-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/d87e0664-ba48-4def-a0e0-2f9f8e736a14">
